src/main/java/com/aws/greengrass/componentmanager/Unarchiver.java

```diff
@@ -38,6 +38,9 @@ public void unarchive(Unarchive method, File toUnarchive, Path unarchiveInto) th
     static void unzip(File zipFile, File destDir) throws IOException {
         try (ZipFile zf = new ZipFile(zipFile)) {
             Enumeration<? extends ZipEntry> entries = zf.entries();
+            // IOUtils uses a 4K buffer by default. Using 64K will make things go faster.
+            byte[] buffer = new byte[1024 * 64];
```

Comment:
Nit- could we use the same constant here so if we change it again later you don't have to remember to change here too

Reply:
Yes, though I don't know that these actually should be coupled. Unarchiving is entirely local, while downloading is over the internet.